### PR TITLE
Logging setup to prepare for demo app

### PR DESCRIPTION
**Pull request by Adovenmuehle**
…fun librato graph along with #count and #measure logs.

Not sure how to write the Procfile so it loads up our example Plug app.

### CODE REVIEW

File: lib/example/example.ex [hunk 1]
```diff
@@ -0,0 +1,24 @@
+defmodule Plexy.Example do
+  use Plug.Router
+  plug Plexy.RequestId
+  plug Plexy.Instrumentor
```
Comment (Adovenmuehle) [inline lib/example/example.ex]: Also not exactly sure how to wrap all the Plexy stuff into one thing so it just becomes `plug Plexy`

Comment (joshwlewis) [inline lib/example/example.ex]: Honestly, I like being able to pick and choose here. Pliny does the same where you the user is in control of which middleware and which order.

Comment (mathias) [inline lib/example/example.ex]: SGTM

**Comment by mathias** [comment]
If we clean this up now that https://github.com/heroku/plexy-demo is broken out, it will just be changes to `config/config.exs` and `lib/plexy/logger.ex` ?

Do we want the changes to `config/config.exs` to set up the logger to be something we tell people to set up in the README?

**Comment by mathias** [comment]
Renamed to indicate what's needed here. This is ready to merge, but we'll need to update https://github.com/heroku/plexy-demo/blob/master/mix.exs#L37 and re-deploy the demo app as part of this merging.

**Comment by mathias** [comment]
Merged but branch was not deleted for plexy-demo's sake.

**Pull request merged**